### PR TITLE
add links to gene specific coverage from chanjo

### PR DESCRIPTION
**Pull request by robinandeer**
Adds links to gene coverage from gene panel page and from variant page

### CODE REVIEW

**Comment by robinandeer** [comment]
if this is overview-able then we can merge
otherwise I'll try to get @vwirta request about a list of poorly covered gene into this PR

File: scout/core/utils.py [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
 from __future__ import absolute_import, unicode_literals
 
-from flask import abort, flash
+from flask import abort, flash, url_for
 from mongoengine import DoesNotExist
 
 from scout.extensions import store
 
 
 def validate_user(current_user, institute_id):
```
Comment (moonso) [inline scout/core/utils.py]: Some indenting left :haircut:

File: scout/core/utils.py [hunk 2]
```diff
@@ -1,21 +1,36 @@
...
-    return abort(403)
+    if institute not in current_user.institutes:
+        flash('You do not have access to this institute.')
+        return abort(403)
 
-  return institute
+    return institute
+
+
+def genecov_links(individuals, hgnc_symbols=None):
```
Comment (moonso) [inline scout/core/utils.py]: Nice

**Comment by moonso** [comment]
Nice! This will be a well appreciated feature. Could we also add links from the variant level to the same gene coverage report?

**Comment by moonso** [comment]
:+1:

**Pull request merged**